index.html

```diff
@@ -67,178 +73,248 @@ <h2>Software Developer</h2>
+                            <div class="verticale"></div>
+                        </li>
+                        <li>
+                            <h3>Ruby</h3>
+                        </li>
+                    </ul>
+                </div>
+
+                <button type="button">See Project </button>
             </div>
+        </div>
 
-            <button type="button">See Project </button>
+        <div class="project fourth">
+            <div class="preview f4"></div>
+            <div class="comment-section4">
+                <h2 class="description_title">Multi-Post Stories</h2>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.
+                    has been the
+                    industry's standard
+                    dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <div class="recipient">
+                    <ul>
+                        <li>
+                            <div>
+                                <h3>css</h3>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="verticale"></div>
+                        </li>
+                        <li>
+                            <div>
+                                <h3>html</h3>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="verticale"></div>
+                        </li>
+                        <li>
+                            <div>
+                                <h3>bootstrap</h3>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="verticale"></div>
+                        </li>
+                        <li>
+                            <h3>Ruby</h3>
+                        </li>
+                    </ul>
+                </div>
 
+                <button type="button">See Project </button>
+            </div>
         </div>
 
 
     </section>
     <section class="form">
-        <h3>CONTACT US</h3>
+        <h3 class="contact-title-phone-v">CONTACT US</h3>
+        <h3 class="contact-title-desktop-v">Let's chat</h3>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that
             needs
             coding. I’d love to help with it</p>
-        
+
         <form action="https://formspree.io/f/moqrdewo" method="post">
-            <input type="text" name ="full name" placeholder="Full name" maxlength="30" required>
+            <input type="text" name="full name" placeholder="Full name" maxlength="30" required
+                class="name-field field1">
+            <input type="text" name="full name" placeholder="Last name" maxlength="30" required
```

Comment:
- [ ] Let's make sure we only have full name only, I know it might be confusing due to Figma design but it was said we should only have Full name field even in the Desktop version 

![image](https://user-images.githubusercontent.com/87689547/155540073-6ba4db6c-a864-4928-b5f9-cda5be1c46d2.png)